### DISCUSSION TITLE
Length of meetups

### DISCUSSION THREAD

**Discussion by AngelOnFira** [rust-gamedev/wg#122] [General]
I was chatting with @cwfitzgerald today about the length of the monthly Rust Gamedev Meetup. Today's meetup was 2.5 hours long, while the last several have been from 1 hour to 1.5 hours. We were discussing how after such a long meetup, it can be hard to get people to keep watching towards the end and then join a Discord call after for a social hangout.

There are various solutions to consider. First, we have the option of restricting people to present within a specific amount of time, and not letting *anyone* go over this time limit. Say 15 minutes. My problem with this is that sometimes there is a lot to discuss on a topic, and it's worth asking more questions and discussing further.

Another solution is to restrict how often someone can present, and favor people who haven't presented in the longest amount of time. That way, maybe some people can go longer, but we could cap off the meetup once we've gone through a few people. The downside of this is that we see many projects using the meetup as motivation to get work done to show off, and this is a good accountability method.

I would also venture further and say that the meetup's value is only half during the live session, and then half in the future when people can watch it on-demand with timestamps for sections they want to watch. This means that the developers have a platform that they can link to with a timestamp, and viewers that might have been busy can consume the content then. So maybe it's not too important to see high numbers of viewers throughout the stream? Unsure about this.

Would love to hear what others think!

**Comment by Ralith** [comment]
I think firmly restricting presentation length, setting a maximum number of slots, and giving preference to those who have least recently presented when the maximum is reached would be a good solution. Extended ad-hoc discussion can and should happen separately, though presenters could leave time for Q&A within their fixed slot budget if they choose.

**Comment by zetanumbers** [comment]
Weekly meetups woohoo!
